ci-operator/step-registry/hypershift/agent/create/hypershift-agent-create-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: hypershift-agent-create
+  steps:
+  - ref: hypershift-agent-create-hostedcluster
```

Comment:
I think that `hypershift-agent-create-hostedcluster` and `hypershift-agent-create-agent` duplicate this:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/baremetal/operator/hypershift/assisted-baremetal-operator-hypershift-commands.sh

Reply:
After we checked [assisted-baremetal-operator-hypershift-commands.sh](https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/baremetal/operator/hypershift/assisted-baremetal-operator-hypershift-commands.sh), we found it also created lots of CR related to hive, clusterdeployment etc. Those will not be used by hypershift conformation test and E2E functional test. If any case failed, it will be hard for us to debug. That's why we created 2 steps here.

Reply:
Got it, if the main goal is to run the conformance test then it makes sense.
If you like to get good coverage for the workers lifecycle I recommend using the operator workflow because hive and its webhooks have some impact on it.